Client/src/app/components/historyComponent/history.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="history-title">Watch History</div>
+
+<div class="history-list" *ngIf="historyReady">
```

Comment:
If you want to get fancy (and I think you should here), Angular has the async pipe for this scenarios, I think you can use it instead of this ngIf: [](https://angular.io/api/common/AsyncPipe)